RetroBar/Languages/português.xaml

```diff
@@ -0,0 +1,56 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:s="clr-namespace:System;assembly=mscorlib">
+
+    <s:String x:Key="retrobar_properties">Propriedades da RetroBar</s:String>
+    <s:String x:Key="taskbar_options">Barra de tarefas</s:String>
+    <BitmapImage x:Key="properties_95" UriSource="../Resources/pt/properties.png" />
+    <BitmapImage x:Key="properties_95_noclock" UriSource="../Resources/pt/properties_noclock.png" />
+    <s:String x:Key="autostart">Iniciar automaticamente no _logon</s:String>
+    <s:String x:Key="language_text">I_dioma:</s:String>
+    <s:String x:Key="language_tip">Selecionar o idioma a ser usado.</s:String>
+    <s:String x:Key="theme_text">_Tema:</s:String>
+    <s:String x:Key="theme_tip">Instale temas na pasta Themes.</s:String>
+    <s:String x:Key="location_text">_Local:</s:String>
+    <s:String x:Key="location_tip">Mudar o local da barra de tarefas na tela.</s:String>
+    <x:Array x:Key="location_values" Type="s:String">
+        <s:String>Esquerda</s:String>
+        <s:String>Superior</s:String>
+        <s:String>Direita</s:String>
+        <s:String>Inferior</s:String>
+    </x:Array>
+    <s:String x:Key="allow_font_smoothing">_Permitir suavização das fontes</s:String>
+    <s:String x:Key="collapse_tray_icons">Ocultar ícones _inativos</s:String>
```

Comment:
Hmm, this is fine since as you said, you're using the XP strings for this one...